### ISSUE TITLE
Store credentials in repo

### ISSUE THREAD

**Issue by internetimagery** [DanielDent/git-annex-remote-rclone#17]
I came across this [in the docs.](https://github.com/ncw/rclone/blob/b2a4ea9304644c6ed2a7f541562ecb93b79153c9/docs/content/docs.md#environment-variables)

Perhaps it would be possible to shuttle config options from annex's built in method into rclone. I assume this would require "initremote" setup to have matching key=value entries.

**Comment by DanielDent** [comment]
Thanks for the note. Yes this looks realistic.

One simple implementation could be to treat key=value entries with keys beginning with RCLONE_ as opaque values which are passed as environment values to rclone when rclone is called.

This would support both this use case and would also have applicability to #3 (some consideration should be given to how these two issues interact - I think having local configuration in the .git config file override the repo-wide configuration would is fine, which would work because rclone environment variables merely set defaults).

I don't personally have time to implement this at the moment, but I would make time to review and merge a pull request.

**Comment by internetimagery** [comment]
Yeah I'll take a crack at it. Though it does seem we cannot use this feature till the next version... damnit, 😜 

https://github.com/ncw/rclone/issues/1227

**Comment by DanielDent** [comment]
Looks like 1.36 is out... :)

**Comment by internetimagery** [comment]
Oh nice! Time to have a look-see. Thanks. 👍
